README.md

```diff
@@ -83,6 +85,15 @@ below to listen to these events.
 This method closes the picker without the user's action.
 Make sure you are not ruining user experience unnecessarily.
 
+#### `simplepicker.reset(date)`:
+
+The reset allows to change the date of the simplepicker example:
+```javascript
+const sp = new SimplePicker();
+sp.reset(new Date(2019, 12, 31, 7, 0, 0));
+sp.open();
+```
+
```

Comment:
Let's add a warning: "**Note**: The date user may have picked would be lost." or similar because of the effect this has on the UX.

Reply:
Not sure how to fix this and resubmit, so I will cancel the pull request and create a new one.